### DISCUSSION TITLE
Failed to load schema(s). fetch is not defined.

### DISCUSSION THREAD

**Discussion by haraldsperre** [0no-co/gql.tada#307] [Q&A] [answered]
I have been having trouble generating the `graphql-env.d.ts` file, and then I recently found out about the cli. Running `npx gql.tada doctor` outputs:
>  ⚠ Unexpected Error 
> Failed to load schema.
> ┗ CombinedError: [Network] fetch is not defined
>       at makeErrorResult (file:///<project root>/node_modules/gql.tada/node_modules/@gql.tada/internal/dist/gql-tada-internal.mjs:943:10)
>       at fetchOperation (file:///<project root>/node_modules/gql.tada/node_modules/@gql.tada/internal/dist/gql-tada-internal.mjs:1172:13)
>       at fetchOperation.next (<anonymous>)
>       at file:///<project root>/frontend/node_modules/gql.tada/node_modules/@gql.tada/internal/dist/gql-tada-internal.mjs:445:28

(Edited the discussion because I originally had error message about typescript version, but it turned out I wasn't using workspace version, so that worked out).

**Comment by kitten** [comment, marked as answer]
Which version of Node are you on? We haven't added a compat check for doctor on Node yet, but we do expect fetch to be available, since that's been around without a flag since Node@18

**Reply by haraldsperre** [reply]
Interestingly enough, I was on Node version 16(!), but I then asked my system to use 18, which still didn't work. After installing version 20.13.1, however, it does work. Thanks!
